### PR TITLE
Staticize Poppins font to be compliant with EU privacy

### PR DESCRIPTION
**Pull request by giuliaghisini**

### CODE REVIEW

File: theme/themes/pastanaga/globals/site.overrides [hunk 1]
```diff
@@ -1,3 +1,5 @@
 /*******************************
         Global Overrides
 *******************************/
+
+@import (multiple) '../extras/fonts';
```
Comment (nileshgulia1) [inline theme/themes/pastanaga/globals/site.overrides:5]: @giuliaghisini Shouldn't we import `fonts.less` in https://github.com/plone/volto/blob/staticize_font/theme/themes/pastanaga/extras/main.less#L560 ?

Comment (sneridagh) [reply theme/themes/pastanaga/globals/site.overrides:5]: @giuliaghisini I think @nileshgulia1 is correct, shouldn't we reference it from main.less?

**Comment by giuliaghisini** [comment]
@nileshgulia1 @sneridagh @tiberiuichim moved fonts.less import in main.less

File: theme/themes/pastanaga/globals/site.overrides (hunk 1 above)
Comment (pnicolli) [reply theme/themes/pastanaga/globals/site.overrides]: To put it in site.overrides was my idea in the first place, sorry 😝 
We "removed" if from site.variables, so it felt like the most appropriate place. In addition, it is more connected with how semantic-ui works. I was not a fan of importing from a different folder though, so I'm ok with changing this as you suggest.

**Review by sneridagh — APPROVED** [review]

**Comment by sneridagh** [comment]
Don't know why but the `cy.waitForResourceToLoad` helper has started to blow up with this PR. Pushing some fixes, but I don't get why...

**Review by nileshgulia1 — APPROVED** [review]

**Review by pnicolli — APPROVED** [review]

**Pull request merged**